api/operator_raft_test.go

```diff
@@ -27,33 +28,181 @@ func TestAPI_OperatorRaftGetConfiguration(t *testing.T) {
 
 func TestAPI_OperatorRaftRemovePeerByAddress(t *testing.T) {
 	t.Parallel()
-	c, s := makeClient(t)
-	defer s.Stop()
+	c1, s1 := makeClientWithConfig(t, nil, func(conf *testutil.TestServerConfig) {
+		if conf.Autopilot == nil {
+			conf.Autopilot = &testutil.TestAutopilotConfig{}
+		}
+		conf.Autopilot.ServerStabilizationTime = "1ms"
```

Comment:
This might be a useful default for the test server but I don't know if there are downsides with setting it. Without this, the `WaitForVoting` function would timeout since it would take 10s+ to become voting.